x-hooks/use-api.tsx

```diff
@@ -64,7 +66,7 @@ export default function useApi() {
   
   api.interceptors.request.use(config => {
     config.headers["wallet"] = typeof window !== 'undefined' && sessionStorage.getItem("currentWallet") || ""
-
+    config.headers["network"] = DEFAULT_NETWORK_NAME;
```

Comment:
This is not correct: the network we are in _might_ not be the default network name

Reply:
if you look 6 line above, you will see, the DEFAULT_NETWORK_NAME its a alias to active network from state.